champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/ranger/bow/Volley.java

```diff
@@ -49,14 +50,17 @@ public String[] getDescription(int level) {
                 "Your next shot is instant, and shoots a volley",
                 "of arrows in the direction you are facing",
                 "",
-                "Each arrow will deal <stat>" + damage + "</stat> damage",
+                "Each arrow will deal <stat>" + baseDamage + "</stat> damage",
```

Comment:
shouldnt you be using getDamage here now?

Reply:
ye[